addresses.json

```
@@ -0,0 +1,14 @@
+{
+    "ProxyAdmin": {
+      "rskMainnet": "0x0000000000000000000000000000000000000000",
+      "rskTestnet": "0x0000000000000000000000000000000000000000"
```

```suggestion
      "rskMainnet": "INVALID",
      "rskTestnet": "INVALID"
```

Using a valid zero address is error-prone

---

Good catch. Totally agree.